etcdctl/ctlv3/command/printer.go

```diff
@@ -194,7 +194,8 @@ func makeEndpointHealthTable(healthList []epHealth) (hdr []string, rows [][]stri
 }
 
 func makeEndpointStatusTable(statusList []epStatus) (hdr []string, rows [][]string) {
-	hdr = []string{"endpoint", "ID", "version", "db size", "is leader", "raft term", "raft index", "raft applied index", "errors"}
+	hdr = []string{"endpoint", "ID", "version", "db size", "is leader", "raft term", "raft index",
```

Comment:
maybe leave the errors to the last column?

Reply:
Yes I was thinking about putting `is learner` next to `is leader`. Only concern is users might already have some scripts parsing the output of `etcdctl endpoint status`, changing the ordering of fields is more likely to break backward compatibility.

Reply:
what about user script is using the last column for errors. 
we can document this change in the changlog. This is not a blocker for this pr.
LGTM.

Reply:
@jingyih @WIZARD-CXY Let's put `is learner` next to `is leader`. It's table output, so unlikely people use this format programmatically.

LGTM.

Reply:
Sounds good. Thanks for the suggestion guys:)